PluginsAndFeatures/azure-toolkit-for-intellij/rider/build.gradle

```diff
@@ -39,5 +39,5 @@ test {
 }
 
 publishPlugin {
-    channels publishChanel
+    channels publish_chanel
```

Comment:
Are we using publish on the build server?

Reply:
My previous intentions. Dropped. We need to make a build config for publishing - https://youtrack.jetbrains.com/issue/RIDER-44653